src/site/content/en/blog/app-history-api/index.md

```diff
@@ -260,23 +271,25 @@ And while the App History API is particularly powerful even with a simple event
 
 ### Programmatic Navigation
 
-You can call `appHistory.navigate('/another_page')` from anywhere in your code to cause a navigation which will be handled by the centralized event handler registered on the "navigate" event.
+You can call `appHistory.navigate('/another_page')` from anywhere in your code to cause a navigation which will be handled by the centralized event handler registered on the "navigate" handler.
 This is intended as an improved aggregation of older methods like `location.assign()` and friends, plus the History API's methods `pushState()` and `replacestate()`.
 
 {% Aside %}
 
-These older programmatic methods for changing the URL are all still supported with the App History API and now cause "navigate"—that is, they're also handled centrally.
+These older programmatic methods for changing the URL are all still supported with the App History API and now fire the "navigate" handler.
+That is, they're also handled centrally.
 Their signatures aren't modified in any way (i.e., they won't now return a `Promise`) by this new specification, and we imagine that in an older codebase, they'll be replaced by calls to `.navigate()` over time.
 
 {% endAside %}
 
-The `.navigate()` method returns a `Promise`, so the invoker can wait until the transition is complete (or is rejected due to failure or being preempted by another navigation).
-It also has an optional options object, allowing you to `replace` the current URL, set a new immutable `state` (to be made available via `getState()`), and `info`.
+The `AppHistory.navigate()` method returns a `Promise`, so the invoker can wait until the transition is complete (or is rejected due to failure or being preempted by another navigation).
+It also has an optional options object, allowing you to `replace` the current URL, set a new immutable `state` (to be made available via `AppHistoryEntry.getState()`), and `AppHistoryNavigateEvent.info`.
 
-The `info` option is worth calling out.
+The `info` property is worth calling out.
 It allows you to pass transient information about this specific navigation event into the "navigate" handler.
 This could be useful to, for example, denote a particular animation that causes the next page to appear (the alternative might be to set a global variable or include it as part of the #hash. Both options are a bit awkward).
```

Comment:
```suggestion
This could be useful to, for example, denote a particular animation that causes the next page to appear. (The alternative might be to set a global variable or include it as part of the #hash. Both options are a bit awkward.)
```